test/verbose_sql_logger_test.rb

```diff
@@ -23,4 +29,63 @@ def up
       assert_match("SET lock_timeout TO '70s'", logs[4])
     end
   end
+
+  def test_does_not_logs_in_output
+    SafePgMigrations.stub(:verbose?, false) do
+      stdout, stderr = capture_io { run_migration }
+
+      assert_equal '', stdout
+      assert_equal '', stderr
+    end
+  end
+
+  def test_logs_with_env
+    ENV['SAFE_PG_MIGRATIONS_VERBOSE'] = '1'
+
+    stdout, = capture_io { run_migration }
+
+    refute_equal '', stdout
```

Comment:
Maybe we should assert the first line of log here? To make the test a bit more robust

Reply:
(same for other tests where we assert that we are in verbose mode)

Reply:
done!